### DISCUSSION TITLE
Someway to update an parquet file

### DISCUSSION THREAD

**Discussion by Guih-sistemas** [G-Research/ParquetSharp#485] [Q&A]
I need to increase by rows, i will select 1000 rows in postgres, and i need to create a file by 1000 rows per time, and increase and updating the file. Final file will have 22000000 rows

**Comment by adamreeve** [comment]
Hi @Guih-sistemas, do you mean you want to be able to close the file in between each write and then re-open and append more rows? Or is it OK to keep the file open and write each batch of 1000 rows as a new row group?

If you want to append to an existing file, this isn't currently possible with ParquetSharp. This has come up before in #213, and the usual approach people use instead is to write separate files or re-write all the data as a new file. But writing 1000 rows at a time as separate row groups would work.
